tensorflow/java/src/main/java/org/tensorflow/types/UInt8.java

```diff
@@ -17,5 +17,8 @@
 
 /** Represents an 8-bit unsigned integer. */
 public class UInt8 {
-  private UInt8() {}
+  private byte value;
```

Comment:
I didn't quite follow the point of these private constructors and value, could you elaborate?

Reply:
This was setting up to support a future generic `scalarValue()` accessor. But I was getting ahead of myself...